apis/supernova/src/handler/data-property-handler.js

```diff
@@ -215,7 +215,7 @@ class DataPropertyHandler {
     measure.qDef = measure.qDef ?? {};
     measure.qDef.qNumFormat = measure.qDef.qNumFormat ?? {};
 
-    if (isEnabled('MASTER_MEASURE_FORMAT')) {
+    if (isEnabled('MASTER_measureURE_FORMAT')) {
```

Comment:
```suggestion
    if (isEnabled('MASTER_MEASURE_FORMAT')) {
```

Reply:
Seems I can't get rid of this mistake :))